### ISSUE TITLE
Rollback: Ignore non-directories under ~/.emacs.d/.cache/.rollback

### ISSUE THREAD

**Issue by microamp** [syl20bnr/spacemacs#4586]
"Rollback Package Update" returns an error when there exists a file (non-directory) found under `~/.emacs.d/.cache/.rollback`, e.g.
```
length: Opening directory: not a directory, /Users/microamp/.emacs.d/.cache/.rollback/.DS_Store/
```

It starts working once I delete the `.DS_Store` file from the directory manually.

**Comment by synic** [comment]
Just got this too.

Good 'ol OS X, adding files you didn't ask it to add.

**Comment by microamp** [comment]
Fixed in #4719
